source/PluginDev/Assets/GooglePlayGames/Editor/GPGSAndroidSetupUI.cs

```diff
@@ -127,5 +157,21 @@ public static bool PerformSetup(string appId, string nearbySvcId)
 
             return true;
         }
+		
```

Comment:
It definitely could.  I wasn't sure exactly what strategy you might have wanted to go with that route, whether to keep them separate (as the Android process was quite dissimilar to this one before these changes).